test/parallel/test-fs-stream-fs-options.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const path = require('path');
+const fs = require('fs');
+const assert = require('assert');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const writeStreamOptions = ['open', 'close', 'write'];
+const readStreamOptions = ['open', 'close', 'read'];
```

Comment:
```suggestion
const streamOpts = ['open', 'close']
const writeStreamOptions = [...streamOpts, 'write'];
const readStreamOptions = [...streamOpts, 'read'];
```
Just suggestion, it's not a big deal.